Formula/neomutt.rb

```diff
@@ -0,0 +1,37 @@
+class Neomutt < Formula
+  desc "Email reader with support for Notmuch (fast indexing), News (NNTP) and much more"
+  homepage "https://www.neomutt.org/"
+  url "https://github.com/neomutt/neomutt/archive/neomutt-20171215.tar.gz"
+  sha256 "7fb76e99a9f23715ad772ad8f7008c6e2db05eed344817055176c76dbd60c1b5"
+  head "https://github.com/neomutt/neomutt.git"
+
+  depends_on "gettext" => :build
+  depends_on "docbook-xsl" => :build
+
+  depends_on "openssl"
+  depends_on "tokyo-cabinet"
+  depends_on "lmdb"
+  depends_on "gpgme"
+  depends_on "libidn"
+  depends_on "notmuch"
+
+  def install
+    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
+
+    args = %W[
```

Comment:
inline these arguments below since the array is unneeded. see `ufraw` for the proper formatting